phpstan.neon

```diff
@@ -44,5 +39,4 @@ parameters:
                 - tests/Rector/Property/TypedPropertyFromColumnTypeRector/config/non_typed_properties.php
                 - tests/Set/DoctrineORM29Set/config/configured_set.php
 
-        # temporary for tests
-        - '#The path "/\.\./preload\.php" was not found#'
+        - '#New objects with "\$.*" name are overridden\. This can lead to unwanted bugs, please pick a different name to avoid it#'
```

Comment:
Could you resolve these here? We should put this rule into work to control our duplicated names

Reply:
I will try.